frontend/apps/os-shell/src/__tests__/legacy/legacyUiTelemetry.test.ts

```
@@ -0,0 +1,286 @@
+
+import {
+    emitLegacyUiHit,
+    getLegacyUiMetrics,
+    LegacyUiHitPayload,
+    resetLegacyUiMetrics,
+} from '../../telemetry/legacyUiTelemetry';
+
+// Mock sessionStorage
+const mockSessionStorage: Record<string, string> = {};
+beforeEach(() => {
+  Object.keys(mockSessionStorage).forEach((key) => delete mockSessionStorage[key]);
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)